action.yaml

```
@@ -57,39 +55,20 @@ runs:
       uses: actions/setup-node@v4
       with:
         node-version: ${{ inputs.node-version }}
-    - name: Check if input values are correct
-      shell: bash
-      run: |
-        if [ -z "${{ inputs.api_key }}" ] && [ -z "${{ inputs.api-key }}" ]; then
-          echo "Error: Both api_key and api-key are undefined. At least one must be defined."
-          exit 1
-        fi
-    - name: Upload the JUnit files
-      if: ${{ inputs.logs == 'true' }}
-      shell: bash
-      run: |
-        npx @datadog/datadog-ci@${{ inputs.datadog-ci-version}} junit upload \
-          --service ${{ inputs.service }} \
-          --logs \
-          --max-concurrency ${{ inputs.concurrency }} \
-          ${{ inputs.extra-args }} \
-          ${{ inputs.files }}
-      env:
-        DATADOG_API_KEY: ${{ inputs.api_key != '' && inputs.api_key || inputs.api-key }}
-        DATADOG_SITE: ${{ inputs.site != '' && inputs.site || inputs.datadog-site }}
-        DD_ENV: ${{ inputs.env }}
-        DD_TAGS: ${{ inputs.tags }}
+
     - name: Upload the JUnit files
-      if: ${{ inputs.logs != 'true' }}
       shell: bash
       run: |
         npx @datadog/datadog-ci@${{ inputs.datadog-ci-version}} junit upload \
-          --service ${{ inputs.service }} \
           --max-concurrency ${{ inputs.concurrency }} \
+          ${{ inputs.logs == 'true' && '--logs' || '' }} \
+          ${{ inputs.auto-discovery == 'true' && '--auto-discovery' || '' }} \
+          ${{ inputs.ignored-paths != '' && format('--ignored-paths {0}', inputs.ignored-paths) || '' }} \
+          ${{ inputs.service != '' && format('--service {0}', inputs.service) || '' }} \
           ${{ inputs.extra-args }} \
           ${{ inputs.files }}
       env:
-        DATADOG_API_KEY: ${{ inputs.api_key != '' && inputs.api_key || inputs.api-key }}
-        DATADOG_SITE: ${{ inputs.site != '' && inputs.site || inputs.datadog-site }}
+        DATADOG_API_KEY: ${{ inputs.api_key }}
+        DATADOG_SITE: ${{ inputs.site }}
```

nitpick. Could we change this to
```suggestion
        DD_API_KEY: ${{ inputs.api_key }}
        DD_SITE: ${{ inputs.site }}
```
since we want to move again from `DATADOG_` at some point